pages/video/[id].tsx

```diff
@@ -0,0 +1,51 @@
+import { Text, Flex } from "@chakra-ui/react";
+import { useRouter } from "next/router";
+import React from "react";
+import VideoPlayer from "../../components/login/videoPlay";
+import axios from "axios";
+
+type Params = {
+  params: {
+    id: string;
+  };
+};
+
+export async function getStaticProps({ params }: Params) {
+  const id = params.id;
+  return {
+    props: {
+      id,
+    },
+  };
+}
+
+export async function getStaticPaths() {
+  const {
+    data: { items },
+  } = await axios.get(
+    `https://youtube.googleapis.com/youtube/v3/search?part=snippet&maxResults=25&q=javscript%2C%20podcast%2C%20startup&key=${process.env.NEXT_PUBLIC_YOUTUBE_API}&regionCode=in`
```

Comment:
If the URL is repeating you can create a constant and import that

`const URL = "something..."`

Reply:
Yes, that would look nice but the URL is just getting used 3 times. 
So I think, it shouldn't be that big of a problem.